File: Hackerrank/suma_ab.cpp
#include <iostream>
#include <vector>
#include <string>

void sumab() {
    int it;
    std::cin >> it;

    std::vector<int> results;

    for (int i = 0; i < it; ++i) {
        std::string n;
        std::cin >> n;

        int a = n[0] - '0';  // Convert first character to integer
        int b = n[1] - '0';  // Convert second character to integer

        results.push_back(a + b);
    }

    for (int r : results) {
        std::cout << r << std::endl;
    }
}

int main() {
    sumab();
    return 0;
}

File: Hackerrank/suma_ab.py
def sumab():
    it = input('Number of iterations:')
    results = []
    for i in it: 
        n = input("Number:")
        n_s = str(n)
        a = int(n_s[0])
        b = int(n_s[1])
        results.append(a+b)
    for r in results:
        print(r)
        



sumab()
